src/main/java/com/atlassian/db/replica/internal/circuitbreaker/BreakerHandler.java

```diff
@@ -1,36 +0,0 @@
-package com.atlassian.db.replica.internal.circuitbreaker;
-
-import com.atlassian.db.replica.api.SqlCall;
-import com.atlassian.db.replica.spi.circuitbreaker.CircuitBreaker;
-
-import java.sql.SQLException;
-
-public class BreakerHandler {
-    private final CircuitBreaker breaker;
-
-    public BreakerHandler(CircuitBreaker breaker) {
-        this.breaker = breaker;
-    }
-
-    public <T> T handle(SqlCall<T> call) throws SQLException {
-        try {
-            return call.call();
-        } catch (Throwable throwable) {
```

Comment:
Bonus: we no longer catch `Throwable`, which is [a smell](https://rules.sonarsource.com/java/RSPEC-1181):

> Catching either `Throwable` or `Error` will also catch `OutOfMemoryError` and `InternalError`, from which an application should not attempt to recover.

If a `CircuitBreaker` impl chooses to catch one anyway, it's on them, not forced by `db-replica`.

Reply:
We catch and rethrow it, so the smell is not relevant.

Reply:
We catch, pass to SPI and then rethrow it. We dunno what happens in the SPI.
But it's not the point of the change, just a "bonus". Even if all impls behave, then at least SonarLint stops the warnings 😉